packages/camera/camera_avfoundation/example/ios/RunnerTests/ThreadSafeEventChannelTests.m

```diff
@@ -63,4 +63,20 @@ - (void)testSetStreamHandler_shouldDispatchToMainThreadIfCalledFromBackgroundThr
   [self waitForExpectationsWithTimeout:1 handler:nil];
 }
 
+- (void)testEventChannel_shouldBeKeptAliveWhenDispatchingBackToMainThread {
+  XCTestExpectation *expectation =
+      [self expectationWithDescription:@"Completion should be called."];
+  dispatch_async(dispatch_queue_create("test", NULL), ^{
+    FLTThreadSafeEventChannel *channel = [[FLTThreadSafeEventChannel alloc]
```

Comment:
`@autoreleasepool` to release it?

Reply:
No need because the channel defined within the scope of the `dispatch_async`. 

I tested it out and this test fails the previous code and passes the current code. So we should be fine.